src/db.js

```diff
@@ -416,43 +417,67 @@ var Database = {
   deleteJSONWebTokensForUser: function(userId) {
     assert(typeof userId === 'number');
     return this.run(
-      'DELETE FROM jsonwebtoken_to_user WHERE user = ?',
+      'DELETE FROM jsonwebtokens WHERE user = ?',
       [userId]
     );
   },
 
   /**
-   * Insert a JSONWebToken into the database and return it's primary key.
+   * Insert a JSONWebToken into the database
+   * @param {JSONWebToken} token
+   * @return {Promise<number>} resolved to JWT's primary key
    */
   createJSONWebToken: async function(token) {
-    const {keyId, user, publicKey, issuedAt} = token;
+    const {keyId, user, publicKey, issuedAt, payload} = token;
     const result = await this.run(
-      'INSERT INTO jsonwebtoken_to_user (keyId, user, issuedAt, publicKey) ' +
-      'VALUES (?, ?, ?, ?)',
-      [keyId, user, issuedAt, publicKey]
+      'INSERT INTO jsonwebtokens (keyId, user, issuedAt, publicKey, payload) ' +
+      'VALUES (?, ?, ?, ?, ?)',
+      [keyId, user, issuedAt, publicKey, JSON.stringify(payload)]
     );
     assert(typeof result.lastID === 'number');
     return result.lastID;
   },
 
   /**
-   * Get a JWT by it's key id.
+   * Get a JWT by its key id.
+   * @param {string} keyId
+   * @return {Promise<Object>} jwt data
    */
   getJSONWebTokenByKeyId: function(keyId) {
     assert(typeof keyId === 'string');
     return this.get(
-      'SELECT * FROM jsonwebtoken_to_user WHERE keyId = ?',
+      'SELECT * FROM jsonwebtokens WHERE keyId = ?',
       keyId
     );
   },
 
+  /**
+   * Get all known JWTs of a user
+   * @param {number} userId
+   * @return {Promise<Array<Object>>}
+   */
+  getJSONWebTokensByUser: function(userId) {
```

Comment:
I didn't see this function actually being used anywhere. Presumably it will be needed later?

Reply:
It's used in https://github.com/mozilla-iot/gateway/pull/419/files#diff-f1ef7a0af26e5388f461170438e6b4adR20 and https://github.com/mozilla-iot/gateway/pull/419/files#diff-f1ef7a0af26e5388f461170438e6b4adR40